twilight-model/src/guild/invite/invite_type.rs

```diff
@@ -0,0 +1,66 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(from = "u8", into = "u8")]
+pub enum InviteType {
+    Guild,
+    GroupDM,
```

Comment:
this should be renamed to `GroupDm` following rust's naming conventions https://github.com/rust-lang/rfcs/blob/master/text/0430-finalizing-naming-conventions.md#fine-points